test/IdRegistry/IdRegistry.symbolic.t.sol

```diff
@@ -176,84 +176,4 @@ contract IdRegistrySymTest is SymTest, Test {
-                svm.createAddress("to"),
-                svm.createUint256("fromDeadline"),
-                svm.createBytes(65, "fromSig"),
-                svm.createUint256("toDeadline"),
-                svm.createBytes(65, "toSig")
-            );
-        } else if (selector == idRegistry.transferAndChangeRecovery.selector) {
-            args = abi.encode(
-                svm.createAddress("to"),
-                svm.createAddress("recovery"),
-                svm.createUint256("deadline"),
-                svm.createBytes(65, "sig")
-            );
-        } else if (selector == idRegistry.transferAndChangeRecoveryFor.selector) {
-            args = abi.encode(
-                svm.createAddress("from"),
-                svm.createAddress("to"),
-                svm.createAddress("recovery"),
-                svm.createUint256("fromDeadline"),
-                svm.createBytes(65, "fromSig"),
-                svm.createUint256("toDeadline"),
-                svm.createBytes(65, "toSig")
-            );
-        } else if (selector == idRegistry.changeRecoveryAddressFor.selector) {
-            args = abi.encode(
-                svm.createAddress("owner"),
-                svm.createAddress("recovery"),
-                svm.createUint256("deadline"),
-                svm.createBytes(65, "sig")
-            );
-        } else if (selector == idRegistry.recover.selector) {
-            args = abi.encode(
-                svm.createAddress("from"),
-                svm.createAddress("to"),
-                svm.createUint256("deadline"),
-                svm.createBytes(65, "sig")
-            );
-        } else if (selector == idRegistry.recoverFor.selector) {
-            args = abi.encode(
-                svm.createAddress("from"),
-                svm.createAddress("to"),
-                svm.createUint256("recoveryDeadline"),
-                svm.createBytes(65, "recoverySig"),
-                svm.createUint256("toDeadline"),
-                svm.createBytes(65, "toSig")
-            );
-        } else if (selector == idRegistry.verifyFidSignature.selector) {
-            args = abi.encode(
-                svm.createAddress("custodyAddress"),
-                svm.createUint256("fid"),
-                svm.createBytes32("digest"),
-                svm.createBytes(65, "sig")
-            );
-        } else if (selector == idRegistry.bulkRegisterIds.selector) {
-            IIdRegistry.BulkRegisterData[] memory ids = new IIdRegistry.BulkRegisterData[](0);
-            args = abi.encode(ids);
-        } else if (selector == idRegistry.bulkRegisterIdsWithDefaultRecovery.selector) {
-            IIdRegistry.BulkRegisterDefaultRecoveryData[] memory ids =
-                new IIdRegistry.BulkRegisterDefaultRecoveryData[](0);
-            args = abi.encode(ids, address(0));
-        } else if (selector == idRegistry.bulkResetIds.selector) {
-            uint24[] memory ids = new uint24[](0);
-            args = abi.encode(ids);
```

Comment:
note: the createCalldata() cheatcode, by default, considers array sizes of 0, 1, and 2, covering more cases than before.